docs/user/reporting/index.asciidoc

```diff
@@ -154,24 +144,15 @@ For information about granting access to embedded dashboards, refer to <<kibana-
 
 . Click *Share > Embed code*.
 
-. Specify how you want to generate the code:
-
-* To display the current state, select *Snapshot*.
-
-* To display up-to-date changes, select *Saved object*.
+. Specify which parts of the dashboard you want to include: Top menu, query, time filter, and filter bar.
 
-* Select the dashboard components you want to share.
+. Click *Copy embed code*.
 
-* To generate a shortened link, select *Short URL*.
+//* To automatically log in anonymous users when you have multiple authentication providers enabled, select *Public URL*.
+//+
+//NOTE: *Public URL* is available only when anonymous access is configured and your anonymous service account has privileges to access what you want to embed. For more information, refer to <<anonymous-authentication,Anonymous authentication>>.
```

Comment:
These lines can be removed

Reply:
Ok Awesome, I commented them out in case they were still needed, but I'll remove them altogether. Thanks!

Reply:
```suggestion

```